components/cam/src/control/camsrfexch.F90

```diff
@@ -465,11 +466,14 @@ subroutine cam_export(state,cam_out,pbuf)
 !PMA adds gustiness to surface scheme c20181128
 
    do i=1,ncol
+      umb(i)           = state%u(i,pver)
+      vmb(i)           = state%v(i,pver)
+      vmag(i)          = max(1.e-5_r8,sqrt( umb(i)**2._r8 + vmb(i)**2._r8))
       cam_out%tbot(i)  = state%t(i,pver)
       cam_out%thbot(i) = state%t(i,pver) * state%exner(i,pver)
       cam_out%zbot(i)  = state%zm(i,pver)
-      cam_out%ubot(i)  = state%u(i,pver) * vmag_gust(i)
-      cam_out%vbot(i)  = state%v(i,pver) * vmag_gust(i)
+      cam_out%ubot(i)  = state%u(i,pver) * (vmag_gust(i)+vmag(i))/vmag(i)
+      cam_out%vbot(i)  = state%v(i,pver) * (vmag_gust(i)+vmag(i))/vmag(i)
```

Comment:
@polunma : These two new lines using vmag(i) are also causing NBFB answers.

Reply:
No this won't work. I will provide a fix.